### ISSUE TITLE
[BUG] Dry run check in preflight check script for server and client does not take relative path for PACKAGE_PATH, only absolute path or current working directory.

### ISSUE THREAD

**Issue by ericchu0** [NVIDIA/NVFlare#1222]
**Describe the bug**
Preflight check for server and client passes when the script is run inside the package directory. When run one level up, I need to specify the full path to the package; otherwise, the dry run check throws an error. Preflight check takes relative path for overseer package.

**To Reproduce**
1. In my example, `/work` directory contains `server` package
2. `cd server`
3.
```
/work/server# nvflare preflight_check -p .
Checking Package: .
-----------------------------------------------------------------------------------------------------------------------------------
| Checks                          | Problems                                                                         | How to fix |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check overseer running          | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check grpc port binding         | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check admin port binding        | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check snapshot storage writable | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check job storage writable      | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check dry run                   | PASSED                                                                           | N/A        |
-----------------------------------------------------------------------------------------------------------------------------------
```
4. `cd ..`
5.
```
/work# nvflare preflight_check -p /work/server
Checking Package: /work/server
-----------------------------------------------------------------------------------------------------------------------------------
| Checks                          | Problems                                                                         | How to fix |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check overseer running          | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check grpc port binding         | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check admin port binding        | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check snapshot storage writable | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check job storage writable      | PASSED                                                                           | N/A        |
|---------------------------------------------------------------------------------------------------------------------------------|
| Check dry run                   | PASSED                                                                           | N/A        |
-----------------------------------------------------------------------------------------------------------------------------------
```
6.
```
/work# nvflare preflight_check -p server
Checking Package: server
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
| Checks                          | Problems                                                                         | How to fix                                 |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check overseer running          | PASSED                                                                           | N/A                                        |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check grpc port binding         | PASSED                                                                           | N/A                                        |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check admin port binding        | PASSED                                                                           | N/A                                        |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check snapshot storage writable | PASSED                                                                           | N/A                                        |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check job storage writable      | PASSED                                                                           | N/A                                        |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check dry run                   | Can't start successfully: Traceback (most recent call last):
  File "/opt/conda/ | Please check the error message of dry run. |
|                                 | lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(c |                                            |
|                                 | ode, main_globals, None,
  File "/opt/conda/lib/python3.8/runpy.py", line 87, in |                                            |
|                                 |  _run_code
    exec(code, run_globals)
  File "/opt/conda/lib/python3.8/site-pac |                                            |
|                                 | kages/nvflare/private/fed/app/server/server_train.py", line 180, in <module>
    |                                            |
|                                 |  main()
  File "/opt/conda/lib/python3.8/site-packages/nvflare/private/fed/app/s |                                            |
|                                 | erver/server_train.py", line 79, in main
    conf = FLServerStarterConfiger(
  F |                                            |
|                                 | ile "/opt/conda/lib/python3.8/site-packages/nvflare/private/fed/app/fl_conf.py", |                                            |
|                                 |  line 58, in __init__
    configure_logging(workspace)
  File "/opt/conda/lib/py |                                            |
|                                 | thon3.8/site-packages/nvflare/private/fed/utils/fed_utils.py", line 219, in conf |                                            |
|                                 | igure_logging
    assert os.path.isfile(log_config_file_path), f"missing log con |                                            |
|                                 | fig file {log_config_file_path}"
  File "/opt/conda/lib/python3.8/genericpath.py |                                            |
|                                 | ", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be str |                                            |
|                                 | ing, bytes, os.PathLike or integer, not NoneType
                                |                                            |
-------------------------------------------------------------------------------------------------------------------------------------------------------------------

```
7. A similar error happens when checking the client package
```
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
| Checks                                   | Problems                                                                         | How to fix                                 |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check overseer running                   | PASSED                                                                           | N/A                                        |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check primary service provider available | PASSED                                                                           | N/A                                        |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check SP's socket server available       | PASSED                                                                           | N/A                                        |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check SP's GRPC server available         | PASSED                                                                           | N/A                                        |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Check dry run                            | Can't start successfully: Traceback (most recent call last):
  File "/opt/conda/ | Please check the error message of dry run. |
|                                          | lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(c |                                            |
|                                          | ode, main_globals, None,
  File "/opt/conda/lib/python3.8/runpy.py", line 87, in |                                            |
|                                          |  _run_code
    exec(code, run_globals)
  File "/opt/conda/lib/python3.8/site-pac |                                            |
|                                          | kages/nvflare/private/fed/app/client/client_train.py", line 208, in <module>
    |                                            |
|                                          |  main()
  File "/opt/conda/lib/python3.8/site-packages/nvflare/private/fed/app/c |                                            |
|                                          | lient/client_train.py", line 79, in main
    conf = FLClientStarterConfiger(
  F |                                            |
|                                          | ile "/opt/conda/lib/python3.8/site-packages/nvflare/private/fed/app/fl_conf.py", |                                            |
|                                          |  line 206, in __init__
    configure_logging(workspace)
  File "/opt/conda/lib/p |                                            |
|                                          | ython3.8/site-packages/nvflare/private/fed/utils/fed_utils.py", line 219, in con |                                            |
|                                          | figure_logging
    assert os.path.isfile(log_config_file_path), f"missing log co |                                            |
|                                          | nfig file {log_config_file_path}"
  File "/opt/conda/lib/python3.8/genericpath.p |                                            |
|                                          | y", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be st |                                            |
|                                          | ring, bytes, os.PathLike or integer, not NoneType
                               |                                            |
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------


```
**Expected behavior**
I would expect relative path support to be an easy fix.

**Desktop (please complete the following information):**
 - OS: ubuntu 20.04
 - Python Version 3.8.12
 - NVFlare Version 2.2.1

**Comment by YuanTingHsieh** [comment]
@ericchu0 thanks for reporting, I've fixed it in #1234

**Comment by ericchu0** [comment]
@YuanTingHsieh thanks for fixing!
